tests/functional/lib/BackbeatConsumer.js

```diff
@@ -852,17 +852,17 @@ describe('BackbeatConsumer shutdown tests', () => {
         async.series([
             next => {
                 const interval = setInterval(() => {
-                    if (consumer._processingQueue.idle()) {
+                    if (consumer._taskScheduler.idle()) {
```

Comment:
would it make sense to keep the old name `_processingQueue` (which it still is, IMHO 😄) : so we avoid extra changes?